MinecraftClient/ChatBot.cs

```diff
@@ -53,6 +55,39 @@ private McClient Handler
             }
         }
 
+        /// <summary>
+        /// Will be called every ~100ms.
+        /// </summary>
+        /// <remarks>
+        /// <see cref="Update"/> method can be overridden by child class so need an extra update method
+        /// </remarks>
+        public void UpdateInternal()
```

Comment:
How about protecting this method using the `sealed` attribute?
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/sealed